### PR TITLE
Fix suspense & render errors

### PR DESCRIPTION
**Pull request by smyzin**
- added suspense list
- changed a little components
- changed render of card

### CODE REVIEW

File: src/components/LoadingCard.tsx [hunk 1]
```diff
@@ -27,14 +27,16 @@ export default function ({
...
-  subtitle: string
+  title?: string
+  subtitle?: string
 }) {
   const { cardColor } = useContext(CardContext)
 
   return (
     <div className={loaderContainer}>
-      <Title titleColor={cardColor} title={title} subtitle={subtitle} />
+      {!!title && !!subtitle && (
```
Comment (T-Damer) [inline src/components/LoadingCard.tsx:37]: Let's extract this into `const titled = !!title && !!substitle`

File: src/components/proofs/List.tsx [hunk 2]
```diff
@@ -1,31 +1,25 @@
...
+        fallback={
+          <CardTitle
+            title="Loading..."
+            subtitle="Please, wait until I load supported NFTs, it takes time"
+          />
+        }
+      >
+        <ListTitle />
+      </Suspense>
+      <Suspense fallback={<LoadingCard title="" subtitle="" />}>
```
Comment (T-Damer) [inline src/components/proofs/List.tsx:20]: Why do you need to pass `title` and `subtitle` here, if they're optinal in the component

File: src/components/proofs/ProofList.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+          Created ZK proofs are saved in the browser even if you switch wallets.
+        </AccentText>
+      )}
+    </>
+  )
+}
+
+export default function () {
+  return (
+    <Suspense fallback={<LoadingCard title="" subtitle="" />}>
```
Comment (T-Damer) [inline src/components/proofs/ProofList.tsx:75]: If we don't pass title and subtitle at all, will it work?

**Review by T-Damer — APPROVED** [review]
cool, let's keep it :)

File: src/components/proofs/ListTitle.tsx [hunk 4]
```diff
@@ -1,14 +1,11 @@
 import { useSnapshot } from 'valtio'
 import CardTitle from 'components/CardTitle'
 import ProofStore from 'stores/ProofStore'
-import useProofAddressesAvailableToCreate from 'hooks/useProofAddressesAvailableToCreate'
 
-export default function () {
+export default function ({ available }: { available: string[] }) {
```
Comment (T-Damer) [inline src/components/proofs/ListTitle.tsx:5]: Avaliable what? `avaliableToProofs`

**Review by T-Damer — APPROVED** [review]
lgtm

File: src/components/proofs/ListTitle.tsx (hunk 4 above)
Comment (smyzin) [reply src/components/proofs/ListTitle.tsx:5]: Renamed

File: src/components/proofs/ProofList.tsx [hunk 5]
```diff
@@ -1,40 +1,33 @@
...
 import ProofStore from 'stores/ProofStore'
 import ReadyERC721ProofsList from 'components/proofs/ReadyERC721ProofsList'
 import ReadyEmailProof from 'components/proofs/ReadyEmailProof'
 import Scrollbar from 'components/Scrollbar'
 import proofStore from 'stores/ProofStore'
-import useProofAddressesAvailableToCreate from 'hooks/useProofAddressesAvailableToCreate'
 import walletStore from 'stores/WalletStore'
 
-function ProofListSuspended() {
+export default function ({ available }: { available: string[] }) {
```
[the remark this answers is not in the document]
Comment (smyzin) [reply src/components/proofs/ProofList.tsx:15]: Renamed

**Review by MixailE — APPROVED** [review]

**Pull request merged**